File: java/program/Shirt.java
class Shirt extends Clothing {
    private String color;
    private String sleeveType;

    public Shirt(String idProduct, String name, String brand, String price, String size, String material, String gender, String color, String sleeveType) {
        super(idProduct, name, brand, price, size, material, gender);
        this.color = color;
        this.sleeveType = sleeveType;
    }

    // Getter and setter color
    public String getColor() {
        return color;
    }
    public void setColor(String color) {
        this.color = color;
    }
    
    // Getter and setter Sleeve Type
    public String getSleeveType() {
        return sleeveType;
    }
    public void setSleeveType(String sleeveType) {
        this.sleeveType = sleeveType;
    }
}

File: python/program/Shirt.py
#Saya Nur Ainun 2202046 mengerjakan LATIHAN PRAKTIKUM 2 dalam mata kuliah Desain dan Pemrograman Berorientasi Objek 
#untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin. 

from Clothing import Clothing

class Shirt(Clothing):
    __color = ""
    __sleeve_type = ""

    #constructor
    def __init__(self,  color = "", slevee_type= ""):
        self.color = color
        self.slevee_type = slevee_type

    # Getter and Setter color
    def getColor(self):
        return self.__color
    def setColor(self, color):
        self.color = color
    
    # Getter and Setter slevee type
    def getSlevee_type(self):
        return self.__sleeve_type
    def setSlevee_type(self, slevee_type):
        self.__sleeve_type = slevee_type